scripts/rancher/manager-lifecycle.sh

```diff
@@ -18,6 +18,7 @@ rancher_install_withcertmanagerclusterissuer() {
   local replicas=$3
   local hostname=$4
   local clusterissuer=$5
+  local bootstrappassword=$6
```

Comment:
Can we remove it?

It's much safer to have a password randomly generated and only accessible to an admin and using a bootstrap doesn't ease the first login. Like for a Helm install, not recommended to use a plaintext password.

Reply:
I think this does that right? However you were first using the ENV `$BOOTSTRAP_PASSWORD` variable from outside of the method, this just makes it a param

Reply:
Here the problem is we force the bootstrap password, which means if the first login is done later there is the risk someone can do it before (in particular with something easy as "admin").
Much better to leave a randomly generated value and not forcing the bootstrap password during the installation.

Reply:
I might have even a better way of setting up a password, I will test right now and will update you all

Reply:
The way I was trying to do it, doesn't cooperated, we will wait until JIRA gets implemented and then we will be setting all those declaratively. For now we are stuck with API calls